apis/rds/v1alpha1/zz_db_cluster.go

```diff
@@ -0,0 +1,508 @@
+	// Specifying SourceRegion autogenerates a pre-signed URL that is a valid request
+	// for the operation that can be executed in the source AWS Region.
+	PreSignedURL *string `json:"preSignedURL,omitempty"`
+	// The daily time range during which automated backups are created if automated
+	// backups are enabled using the BackupRetentionPeriod parameter.
+	//
+	// The default is a 30-minute window selected at random from an 8-hour block
+	// of time for each AWS Region. To see the time blocks available, see Adjusting
+	// the Preferred DB Cluster Maintenance Window (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.Maintenance.html#AdjustingTheMaintenanceWindow.Aurora)
+	// in the Amazon Aurora User Guide.
+	//
+	// Constraints:
+	//
+	//    * Must be in the format hh24:mi-hh24:mi.
+	//
+	//    * Must be in Universal Coordinated Time (UTC).
+	//
+	//    * Must not conflict with the preferred maintenance window.
+	//
+	//    * Must be at least 30 minutes.
+	PreferredBackupWindow *string `json:"preferredBackupWindow,omitempty"`
+	// The weekly time range during which system maintenance can occur, in Universal
+	// Coordinated Time (UTC).
+	//
+	// Format: ddd:hh24:mi-ddd:hh24:mi
+	//
+	// The default is a 30-minute window selected at random from an 8-hour block
+	// of time for each AWS Region, occurring on a random day of the week. To see
+	// the time blocks available, see Adjusting the Preferred DB Cluster Maintenance
+	// Window (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/USER_UpgradeDBInstance.Maintenance.html#AdjustingTheMaintenanceWindow.Aurora)
+	// in the Amazon Aurora User Guide.
+	//
+	// Valid Days: Mon, Tue, Wed, Thu, Fri, Sat, Sun.
+	//
+	// Constraints: Minimum 30-minute window.
+	PreferredMaintenanceWindow *string `json:"preferredMaintenanceWindow,omitempty"`
+	// The Amazon Resource Name (ARN) of the source DB instance or DB cluster if
+	// this DB cluster is created as a read replica.
+	ReplicationSourceIdentifier *string `json:"replicationSourceIdentifier,omitempty"`
+	// For DB clusters in serverless DB engine mode, the scaling properties of the
+	// DB cluster.
+	ScalingConfiguration *ScalingConfiguration `json:"scalingConfiguration,omitempty"`
+	// SourceRegion is the source region where the resource exists. This is not
+	// sent over the wire and is only used for presigning. This value should always
+	// have the same region as the source ARN.
+	SourceRegion *string `json:"sourceRegion,omitempty"`
+	// A value that indicates whether the DB cluster is encrypted.
+	StorageEncrypted *bool `json:"storageEncrypted,omitempty"`
+	// Tags to assign to the DB cluster.
+	Tags []*Tag `json:"tags,omitempty"`
+	// A list of EC2 VPC security groups to associate with this DB cluster.
+	VPCSecurityGroupIDs       []*string `json:"vpcSecurityGroupIDs,omitempty"`
+	CustomDBClusterParameters `json:",inline"`
```

Comment:
hey @hiteshghia-splunk do we need the CustomDBClusterParameters struct defined here?

Reply:
nv, I see custom_types.go